addons/crm/models/crm_lead.py

```diff
@@ -1224,6 +1224,12 @@ def get_import_templates(self):
             'template': '/crm/static/xls/crm_lead.xls'
         }]
 
+    def _set_demo_create_date(self, create_date):
```

Comment:
If tde is ok with this, it's ok for me.

Reply:
Well that's crappy. Anyway we have to put somewhere some code to update create date. What about having a hook in models.py to allow writing on create date if it is redefined on the model (not only the default log access column but a really defined column) ? It would actually make it work like "before I don't remember which version" where redefining a log access column would grant access to it.